### DISCUSSION TITLE
statistical strategy

### DISCUSSION THREAD

**Discussion by afreitag33** [Gulf-IEA/scale-paper#4] [General]
Our discussion in Santa Barbara led us to some sort of compilation of linear regressions, but we needed to go do some reading on what the options are. We have an overall path model, which in theory is a subset of our larger system conceptual model, but we're adding some factors.

wetland loss --> edge --> landings

Gini index and number/location of fish dealers/processors likely mediate this relationship and then landings are affected by switching in and out of the recreational sector (creel surveys and 6-pack data). How do we incorporate these?

An [SEM model](https://en.wikipedia.org/wiki/Structural_equation_modeling) might help us structure the relationships and are generally underwritten by generalized linear models. In the past, we have also used simple correlations (when we weren't sure of directionality), and since the data is spatial we could do geographically weighted regressions instead. There's also possibilities of incorporating latent variables through SEM or other methods. Thoughts? Alternatives?

**Comment by SeannRegan-NOAA** [comment]
Given some recent conversations, I think that we might want to look into nonlinear relationships perhaps explore (GAMs) rather than SEMs.

**Reply by afreitag33** [reply]
That was our final conclusion. And that we need to focus on the potential for nonlinear relationships, so we do need something like a GAM. We proposed a 3-step process to looking at the stats:
1) Visualize each variable over a time series and try a correlation like the original paper to make sure we see the same effects
2) Linear regressions between connected variables to see if there's potential linear causal linkages between them
3) GAMs or multiple regressions (based on what we think in visualizations in step 1) to incorporate more of the model
